src/sentry/static/sentry/app/views/organizationDiscover/result/utils.jsx

```diff
@@ -41,4 +41,27 @@ const DarkGray = styled.span`
   color: ${p => p.theme.gray5};
 `;
 
-export {getDisplayValue};
+/**
+ * Downloads a Snuba result object as CSV format
+ *
+ * @param {Object} result Result received from Snuba
+ * @param {Object} result.data Result data object from Snuba
+ * @param {String} result.meta Result metadata from Snuba
+ * @returns {Void}
+ */
+export function downloadAsCsv(result) {
+  const {meta, data} = result;
+  const headings = meta.map(({name}) => name);
+  const rows = [
+    headings,
+    ...data.map(row => {
+      return headings.map(col => row[col]);
+    }),
+  ];
+
+  const csvContent = `data:text/csv;charset=utf-8,${rows
+    .map(row => row.join(','))
```

Comment:
Thanks, good point. Added quotes to any cell starting with + - = or @

Reply:
Also values with commas in them. What about values with double quotes? What about newline characters?

I really think you should use a CSV parser instead of this. Or, use the server. Since Python has `csv` in stdlib.

https://en.wikipedia.org/wiki/Comma-separated_values#Basic_rules